cli/src/utils.rs

```diff
@@ -1128,7 +1129,7 @@ pub async fn deploy_payout_batch<S: Middleware + 'static>(
         Ok(receipt) => receipt,
         Err(error) => {
             let date = chrono::offset::Utc::now().to_string();
-            let file_path = PathBuf::from(&format!("./FailedPayouts{}", date));
```

Comment:
ok if I had to guess this is the one we actually saw - the transaction was send but the receipt is an error for some reason? in this case it seems possible that the transaction did in fact get sent, but we ran into an error getting the receipt - seems like it might be possible that the transactions succeeded OR failed, so maybe yet another name - `ReceiptFailedPayouts` or something?